code/modules/surgery/organs/organ_external.dm

```diff
@@ -138,6 +138,30 @@
 	replaced(H)
 	return 1
 
+/obj/item/organ/external/attack(mob/living/carbon/C, mob/user)
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(H.species && H.species.name == "Machine" && src.status == ORGAN_ROBOT)
```

Comment:
Implied `src` here as `status` is a var of the object you're redefining this proc for.